src/apprt/gtk/ui/menu-surface-context_menu.ui

```diff
@@ -0,0 +1,90 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
is it possible to use double quotes here too?

Reply:
Yes.